Translations/Language.zh-cn.xml

```diff
@@ -60,7 +60,7 @@
     <entry lang="zh-cn" key="IDC_PAUSE">暂停(&amp;P)</entry>
     <entry lang="zh-cn" key="IDC_PIM_ENABLE">使用 P&amp;IM</entry>
     <entry lang="zh-cn" key="IDC_NEW_PIM_ENABLE">调整 PIM</entry>
-    <entry lang="zh-cn" key="IDC_QUICKFORMAT">快速格式化</entry>
+    <entry lang="zh-cn" key="IDC_QUICKFORMAT">快速格式化(全0填充)</entry>
```

Comment:
This is not always true: quick format for disks doesn't fill with zeros, it rather leaves the content of disk sectors as it is.
Moreover, for non technical users, it is not always good to add information. Simplifying UI and th reducing the number of information displayed is best.

Reply:
根据我的测试（Windows 11 x64），快速格式化创建的卷大部分内容是0，难道是系统决定的？Linux会发生变化？
如果是这样，我建议翻译为 **简单格式化** ，因为它并不 **快**（其它语言也修改）

Reply:
Indeed, when you quick format a file container, most of its content might be filled with zeros.

However, it's essential to differentiate between file containers and disks. Quick formatting a disk or partition volume does not fill it with zeros. Instead, it simply leaves the existing content of the disk sectors as they are. In the context of Windows, a "quick format" essentially sets up a new file system (like NTFS or FAT32) and constructs a new empty file table, but does not overwrite or erase the volume's existing data. This is why it's faster than a full format. The underlying data remains intact until it gets overwritten by new data at a later point in time. On Linux, the behavior of quick formatting is analogous.

Regarding your suggestion to term it as "simple formatting" — "quick format" is a well-established and recognized term in the IT industry. Users, both technical and non-technical, generally understand its implications. In VeraCrypt, our quick format functionality aligns with the quick format procedures in both Windows and Linux. It's vital to maintain terminological consistency to avoid confusion and ensure clarity for users familiar with the term.

Reply:
在国际上是公认的术语，但在中国普通用户里是需要搜索才能理解的；我理解保持术语一致性的重要性。

Reply:
> Indeed, when you quick format a file container, most of its content might be filled with zeros.
> 
> However, it's essential to differentiate between file containers and disks. Quick formatting a disk or partition volume does not fill it with zeros. Instead, it simply leaves the existing content of the disk sectors as they are. In the context of Windows, a "quick format" essentially sets up a new file system (like NTFS or FAT32) and constructs a new empty file table, but does not overwrite or erase the volume's existing data. This is why it's faster than a full format. The underlying data remains intact until it gets overwritten by new data at a later point in time. On Linux, the behavior of quick formatting is analogous.
> 
> Regarding your suggestion to term it as "simple formatting" — "quick format" is a well-established and recognized term in the IT industry. Users, both technical and non-technical, generally understand its implications. In VeraCrypt, our quick format functionality aligns with the quick format procedures in both Windows and Linux. It's vital to maintain terminological consistency to avoid confusion and ensure clarity for users familiar with the term.

The term 'Quick formatting' has been widely used in Windows for decades, and its corresponding actual meaning has a strict directional effect from beginning to end (for professionals, you can see how it will work; for non professionals, they can also expect it to play the same role as in Windows or even Android). Inventing new words to describe a mature concept that has already been used in the industry will only lead to broader misunderstandings.